fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/IntelliJ.scala

```diff
@@ -87,6 +87,13 @@ object IntelliJ {
       "--",
       V.bloopVersion
     )
+    newJson("argvExit") = List[String](
```

Comment:
If the goal is restarting the server, is this the best name? It's an implementation detail of Bloop that restarting is done via exit with the next command starting a new server. Should this be calle `argvRestart`?

Is this something that's only relevant for Fastpass and Bloop?

Reply:
It's relevant to fastpass and scalaPlugin. The change to scala plugin is not accepted yet, so that's why it's only a draft PR

argvExit is IMO better, because `bloop exit` command is expected to be there